### PR TITLE
OP-1032 Script updates for 1.12.1

### PR DESCRIPTION
**Pull request by mizzioisf**
New batch of script updates, should be ready for 1.12.1.

- More "interactive": get back to menu instead of exiting where possible
- Better and safer clean options (it is possible to select what to delete)
- Added direct database settings reading from database.properties file
- Updates and fixes
- Show OH version in main menu

_needs_testing_ !

### CODE REVIEW

File: oh.ps1 [hunk 1]
```diff
@@ -269,12 +269,81 @@ function script_menu {
 }
 
 ###################################################################
-function get_confirmation {
+function get_confirmation ($arg) {
 	$choice = Read-Host -Prompt "(y/n) ? "
```
Comment (dbmalkovsky) [inline oh.ps1:273]: Nit pick:  why are there 19 occurrences in this PR where there is a space before the question mark?   That isn't typically how a question is written.

File: oh.ps1 [hunk 2]
```diff
@@ -887,61 +921,49 @@ function write_config_files {
...
-	$filetodel="$OH_PATH\$OH_DIR\rsc\database.properties.old"; if (Test-Path $filetodel -PathType leaf) { Remove-Item $filetodel -Recurse -Confirm:$false -ErrorAction Ignore }
-	$filetodel="$OH_PATH\$OH_DIR\rsc\log4j.properties"; if (Test-Path $filetodel -PathType leaf) { Remove-Item $filetodel -Recurse -Confirm:$false -ErrorAction Ignore }
-	$filetodel="$OH_PATH\$OH_DIR\rsc\log4j.properties.old"; if (Test-Path $filetodel -PathType leaf) { Remove-Item $filetodel -Recurse -Confirm:$false -ErrorAction Ignore }
-	$filetodel="$OH_PATH\$OH_DIR\rsc\dicom.properties"; if (Test-Path $filetodel -PathType leaf) { Remove-Item $filetodel -Recurse -Confirm:$false -ErrorAction Ignore }
-	$filetodel="$OH_PATH\$OH_DIR\rsc\dicom.properties.old"; if (Test-Path $filetodel -PathType leaf) { Remove-Item $filetodel -Recurse -Confirm:$false -ErrorAction Ignore }
-	$filetodel="$OH_PATH\$OH_DIR\$LOG_DIR\*"; if (Test-Path $filetodel -PathType leaf) { Remove-Item $filetodel -Recurse -Confirm:$false -ErrorAction Ignore }
+function clean_database {
+	# kill mysqld zombies
+	Write-Host "Killing mysql processes..."
```
Comment (dbmalkovsky) [inline oh.ps1:926]: another minor point
```suggestion
	Write-Host "Killing mariadb/mysql processes..."
```

File: oh.ps1 (hunk 1 above)
Comment (mizzioisf) [reply oh.ps1]: Thank you david, fixed

File: oh.ps1 (hunk 2 above)
Comment (mizzioisf) [reply oh.ps1]: Fixed.

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**